student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,5 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(),"vido2373@colorado.edu", "Usernames do not match\n");
```

Comment:
@vido2373  you should verify your conffile string using `malloc_username_from_conf_file` as mentioned in the TODO, rather than hard-coding your username.